taskcluster/scripts/toolchain/compile_qt_6.ps1

```diff
@@ -74,6 +83,75 @@ Set-Location $FETCHES_PATH/qt-everywhere-src-$QT_VERSION
   -qt-zlib  `
   -openssl-runtime `
   -prefix $BUILD_PREFIX `
+} else {
+  # For newer qt versions, let's trim what we dont need.
+  ./configure.bat `
+  -static  `
+  -opensource  `
+  -debug-and-release `
+  -confirm-license  `
+  -silent  `
+  -make libs  `
+  -no-feature-dynamicgl `
```

Comment:
Should we use the following which were used in 6.2?
  -nomake tests  `
  -nomake examples  `